### ISSUE TITLE
DLNA option 😍

### ISSUE THREAD

**Issue by JamesDAdams** [ironsmile/euterpe#43]
Hi, your soft is very good, and an option to cast in dlna on speakers would be even better :)

**Comment by ironsmile** [comment]
Hello! Yea, you are right. I've been thinking about that too. But two reasons have stopped to do it so far. First, I don't really use any DLNA devices and come to think of it I don't think I have any. It should be a problem, though. The biggest issue is I haven't decided whether it makes sense for the server to do it or for one (all?) of it clients to implement this. It would be great if the server could do it but on the other hand I expect it to be on a different network and possibly accessed via Internet. This is what I do. The other option is adding a DLNA server to the clients (mobile, GTK, Rhythmbox plug-in) which are presumably in the same network as the DLNA speakers. What stopped me from going forward is that it is a duplication of effort for every client. And all of them still require other features to be added before I get to DLNA.

TL;DR - I want to work on this too. But there are other things with higher priority for me at the moment.
